### PR TITLE
Feature/use rewrite to file import

### PR DESCRIPTION
**Pull request by SindreSvendby**
- [X] **feature** _(A new feature_)

This feature rewrites shiny so we can  referer to the `src/index.js` even without having a working  treeshaking going. Because it seems it is very hard to get to work, I think impossible if you are not on webpack 4. Instead babel transform all the ref. to the file they are coming from. 

Unfortunally this means adding `"import/prefer-default-export": 0,`  since we need to rename import to different files. 

See https://material-ui.com/guides/minimizing-bundle-size/  for how matarial-ui has done it. 
We should keep an eye on how they fix it in the future, but now, I think this is the best approche.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "rules": {
     "react/jsx-filename-extension": 0,
     "react/jsx-curly-brace-presence": 0,
-		 "import/prefer-default-export": 0,
+     "import/prefer-default-export": 0,
+     "import/no-duplicates": 0,
```
Comment (misund) [inline .eslintrc.json]: Is there something weird with the indentation in this file?

File: babel-tranform-shiny.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+		return mapping[importName]
+	}
+	throw new Error(importName + ' is not in mapping')
+}
+
+function getMapping() {
+	return {
+		themes: '../themes',
+        HorizontalOverflowGradient: '../atoms/HorizontalOverflowGradient',
+        ShinyThemeProvider: '../atoms/ShinyThemeProvider',
```
Comment (misund) [inline babel-tranform-shiny.js]: indentation

File: package.json [hunk 3]
```diff
@@ -62,6 +62,7 @@
 		"babel-eslint": "8.2.3",
 		"babel-plugin-react-docgen": "2.0.0-rc.0",
 		"babel-plugin-styled-components": "1.5.1",
+		"babel-plugin-transform-imports": "^1.5.0",
```
Comment (misund) [inline package.json]: yarn add -E (exact version number)

File: src/atoms/LinkBarButtonBase.js [hunk 4]
```diff
@@ -1,7 +1,5 @@
-import {
-	BlockLink,
-	LinkBarLinkBase,
-} from '..';
+import {	BlockLink	} from '..';
+import {	LinkBarLinkBase	} from '..';
```
Comment (misund) [inline src/atoms/LinkBarButtonBase.js]: Why didn't eslint catch this? :-/

Comment (SindreSvendby) [reply src/atoms/LinkBarButtonBase.js]: Fixed, but eslint did not care. as long as it's one space. it do not complaint.

**Pull request merged**